src/components/Review/Review/Review.tsx

```diff
@@ -12,26 +12,23 @@ import { nanoid } from 'nanoid'
 interface IReview {
   isUserReview: boolean
   review: ReviewType
-  toggleReviewExpansion?: (id: string) => void
   deleteReview?: (id: string) => void
   rateReview?: (id: string, isLike: boolean) => void
-  isExpanded?: boolean
 }
 
 const Review: React.FC<Readonly<IReview>> = ({
   isUserReview = false,
   review,
-  toggleReviewExpansion = () => {},
-  deleteReview = () => {},
-  rateReview = () => {},
-  isExpanded,
+  deleteReview = () => { },
+  rateReview = () => { },
 }) => {
+  const [isSeeMoreExpanded, setIsSeeMoreExpanded] = useState(false)
```

Comment:
Hello! Thank you very much for your pull request!

I think it would be better to rename isSeeMoreExpanded to something like isReviewExpanded or isShowMoreClicked, and toggleReviewExpansionHandler to something like seeMoreButtonClickHandler.

Reply:
Hello @anksuunamun, Thanks for reviewing.
changed the naming conventions accordingly. can you take a loot at it?